pmd-apex/src/main/resources/rulesets/apex/ruleset.xml

```diff
@@ -419,4 +419,15 @@
          <property name="cc_block_highlighting" value="false" />
       </properties>
    </rule>
+
+   <!-- DOCUMENTATION -->
+   <rule ref="category/apex/documentation.xml/ApexDoc" message="Document classes, methods, and properties that are public or global.">
+      <priority>3</priority>
+      <properties>
+         <!-- relevant for Code Climate output only -->
+         <property name="cc_categories" value="Style" />
+         <property name="cc_remediation_points_multiplier" value="50" />
```

Comment:
What should the multiplier be?

Reply:
See here for explanation of remediation points: https://github.com/codeclimate/spec/blob/master/SPEC.md#remediation-points

In the implementation of our CodeClimateRenderer, we use the baseline of 50_000 (which is the estimated required time for a trivial fix) and multiply it with the property defined here: https://github.com/pmd/pmd/blob/master/pmd-core/src/main/java/net/sourceforge/pmd/renderers/CodeClimateRenderer.java#L124-L132

I see, that we use multipliers between 1 and 250. Fixing missing documentation can be time consuming and we provide here a rough guess across all types of missing doc (class documentation might take more effort than a property documentation).

IMHO, 50 sounds like a reasonable value to me.